py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -319,3 +322,91 @@ def compile_module(
         settings.use_fast_partitioner = True
 
     return partitioned_module
+
+
+def convert_method_to_trt_engine(
```

Comment:
- Rename to `convert_module_to_trt_engine`
- In `torch_tensorrt._compile` - limit usage of `ir="dynamo"` for non-forward method names
- `convert_module_to_trt_engine` in `._compile`, `convert_module_to_trt_engine` in `dynamo._compiler.py`, and `convert_method_to_trt_engine`

Reply:
Should this still be `convert_module_to_trt_engine` @narendasan?

Reply:
Use `dynamo.convert_module_to_trt_engine`, but keep top-level function as `convert_method_to_trt_engine` above Dynamo/TS scope (`_compiler.py`)

Reply:
> Use `dynamo.convert_module_to_trt_engine`, but keep top-level function as `convert_method_to_trt_engine` above Dynamo/TS scope (`_compiler.py`)

Do we need to change `ts.convert_method_to_trt_engine` to `ts.convert_module_to_trt_engine` accordingly?

Reply:
I think that should not be necessary, since the `ts` variant can convert arbitrary methods.